projects/1.21/assets/rope-bridge/ropebridge/lang/zh_cn.json

```
@@ -0,0 +1,55 @@
+{
+  "advancement.ropebridge.build_bridge.desc": "使用新的绳桥枪搭建一座绳桥",
+  "advancement.ropebridge.build_bridge.title": "绳桥",
+  "advancement.ropebridge.build_ladder.desc": "使用新的梯子枪搭建一架绳梯",
+  "advancement.ropebridge.build_ladder.title": "绳梯",
+  "advancement.ropebridge.craft_bridge_builder.desc": "用绳桥锚、枪管和枪托制作绳桥枪",
+  "advancement.ropebridge.craft_bridge_builder.title": "绳桥枪",
+  "advancement.ropebridge.craft_ladder_builder.desc": "用梯子钩、枪管和枪托制作梯子枪",
+  "advancement.ropebridge.craft_ladder_builder.title": "梯子枪",
+  "advancement.ropebridge.root.desc": "轻松搭建桥梁！",
+  "advancement.ropebridge.root.title": "索桥",
+  "block.ropebridge.acacia_bridge": "金合欢绳桥",
+  "block.ropebridge.acacia_rope_ladder": "金合欢绳梯",
+  "block.ropebridge.bamboo_bridge": "竹绳桥",
+  "block.ropebridge.bamboo_rope_ladder": "竹绳梯",
+  "block.ropebridge.birch_bridge": "白桦绳桥",
+  "block.ropebridge.birch_rope_ladder": "白桦绳梯",
+  "block.ropebridge.cherry_bridge": "樱花绳桥",
+  "block.ropebridge.cherry_rope_ladder": "樱花绳梯",
+  "block.ropebridge.crimson_bridge": "绯红绳桥",
+  "block.ropebridge.crimson_rope_ladder": "绯红绳梯",
+  "block.ropebridge.dark_oak_bridge": "深色橡木绳桥",
+  "block.ropebridge.dark_oak_rope_ladder": "深色橡木绳梯",
+  "block.ropebridge.jungle_bridge": "丛林绳桥",
+  "block.ropebridge.jungle_rope_ladder": "丛林绳梯",
+  "block.ropebridge.mangrove_bridge": "红树林绳桥",
+  "block.ropebridge.mangrove_rope_ladder": "红树林绳梯",
+  "block.ropebridge.oak_bridge": "橡木绳桥",
+  "block.ropebridge.oak_rope_ladder": "橡木绳梯",
+  "block.ropebridge.spruce_bridge": "云杉绳桥",
+  "block.ropebridge.spruce_rope_ladder": "云杉绳梯",
+  "block.ropebridge.warped_bridge": "诡异绳桥",
+  "block.ropebridge.warped_rope_ladder": "诡异绳梯",
+  "chat.ropebridge.info.bad_side": "无法在方块的%f面建造绳梯，请尝试侧面！",
```

```suggestion
  "chat.ropebridge.info.bad_side": "无法在方块的%f建造绳梯，请尝试侧面！",
```
与下方L42、L43联动，应该是把“底部”“顶部”放到`%f`的位置
~~不知道为什么不用`%s`~~